docs/modelfile.md

```diff
@@ -42,6 +42,37 @@ INSTRUCTION arguments
 
 ## Examples
 
+### `Modelfile`s in [ollama.ai/library][1]
```

Comment:
Let's move this to the bottom of the doc under `### Viewing the Modelfile for a model`

Reply:
Okay, just did it, thanks!